src/main/scala/advent/solutions/Day3.scala

```diff
@@ -64,7 +233,26 @@ object Day3 {
       *         or [[None]] if they don't intersect at all
       */
     def distanceToIntersection(wire0: Wire, wire1: Wire): Option[Int] = {
-      ???
+      val wire0Coordinates: List[Coordinate] =
```

Comment:
This could be a function `allCoordinates`, which takes a `Wire` and returns a `List[Coordinate]`.  If that list didn't include the starting coordinate (0,0), we wouldn't need to filter it out later.